### PR TITLE
Fk optimus macaque

### PR DESCRIPTION
**Pull request by kevinpalis**
- Changed sctools to accept mitochondrial gene names
- Readme updated for Optimus
- Switching sctools docker image from Quay to the latest one in GCR

### CODE REVIEW

**Review by khajoue2 — APPROVED** [review]
Looks good

File: pipelines/skylab/optimus/Optimus.changelog.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 5.6.0
+2022-08-19 (Date of Last Commit)
```
Comment (timaeusx) [inline pipelines/skylab/optimus/Optimus.changelog.md:2]: You may want to update this date!

File: pipelines/skylab/optimus/Optimus.changelog.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# 5.6.0
```
[the remark this answers is not in the document]
Comment (kevinpalis) [reply pipelines/skylab/optimus/Optimus.changelog.md:1]: Sorry about that. Removed.

File: pipelines/skylab/optimus/Optimus.changelog.md (hunk 1 above)
Comment (kevinpalis) [reply pipelines/skylab/optimus/Optimus.changelog.md]: Done.

**Review by nikellepetrillo — APPROVED** [review]

**Pull request merged**